internal/database/models.go

```diff
@@ -1,160 +0,0 @@
-type Network struct {
-	ID          int64  `db:"network_id,omitempty"`
-	NetworkName string `db:"network_name"`
-	NetworkType string `db:"network_type"`
-	IP          string `db:"ip"`
-	ASN         uint   `db:"asn"`
-	CountryCode string `db:"network_country_code"`
-}
-
-// URL represents URLs from the testing lists
-type URL struct {
-	ID           sql.NullInt64  `db:"url_id,omitempty"`
-	URL          sql.NullString `db:"url"`
-	CategoryCode sql.NullString `db:"category_code"`
-	CountryCode  sql.NullString `db:"url_country_code"`
-}
-
-// Measurement model
-type Measurement struct {
-	ID               int64          `db:"measurement_id,omitempty"`
-	TestName         string         `db:"test_name"`
-	StartTime        time.Time      `db:"measurement_start_time"`
-	Runtime          float64        `db:"measurement_runtime"` // Fractional number of seconds
-	IsDone           bool           `db:"measurement_is_done"`
-	IsUploaded       bool           `db:"measurement_is_uploaded"`
-	IsFailed         bool           `db:"measurement_is_failed"`
-	FailureMsg       sql.NullString `db:"measurement_failure_msg,omitempty"`
-	IsUploadFailed   bool           `db:"measurement_is_upload_failed"`
-	UploadFailureMsg sql.NullString `db:"measurement_upload_failure_msg,omitempty"`
-	IsRerun          bool           `db:"measurement_is_rerun"`
-	ReportID         sql.NullString `db:"report_id,omitempty"`
-	URLID            sql.NullInt64  `db:"url_id,omitempty"` // Used to reference URL
-	MeasurementID    sql.NullInt64  `db:"collector_measurement_id,omitempty"`
-	IsAnomaly        sql.NullBool   `db:"is_anomaly,omitempty"`
-	// FIXME we likely want to support JSON. See: https://github.com/upper/db/issues/462
-	TestKeys            string         `db:"test_keys"`
-	ResultID            int64          `db:"result_id"`
-	ReportFilePath      sql.NullString `db:"report_file_path,omitempty"`
-	MeasurementFilePath sql.NullString `db:"measurement_file_path,omitempty"`
-}
-
-// Result model
-type Result struct {
-	ID             int64     `db:"result_id,omitempty"`
-	TestGroupName  string    `db:"test_group_name"`
-	StartTime      time.Time `db:"result_start_time"`
-	NetworkID      int64     `db:"network_id"`     // Used to include a Network
-	Runtime        float64   `db:"result_runtime"` // Runtime is expressed in fractional seconds
-	IsViewed       bool      `db:"result_is_viewed"`
-	IsDone         bool      `db:"result_is_done"`
-	IsUploaded     bool      `db:"result_is_uploaded"`
-	DataUsageUp    float64   `db:"result_data_usage_up"`
-	DataUsageDown  float64   `db:"result_data_usage_down"`
-	MeasurementDir string    `db:"measurement_dir"`
-}
-
-// PerformanceTestKeys is the result summary for a performance test
-type PerformanceTestKeys struct {
-	Upload   float64 `json:"upload"`
-	Download float64 `json:"download"`
-	Ping     float64 `json:"ping"`
-	Bitrate  float64 `json:"median_bitrate"`
-}
-
-// Finished marks the result as done and sets the runtime
```

Comment:
Please, move the refactored functions from the other file back into this file. By doing that, it will become super obvious from the diff that we're just applying small changes and the diff will basically vouch for its own correctness in a much more easy and obvious way. I understand why you wanted to delete this file and the change in itself makes sense. However, by doing this change now, you're making a review for correctness of this diff much more complicated for a human to perform.